### ISSUE TITLE
No results for WoP

### ISSUE THREAD

**Issue by chmercesmoreira** [Jackett/Jackett#1592]
Jackett version:  0.7.1594
Mono version (if not using Windows):  mono 5.0.0.100

-------------------------------

I'm getting these error messages when trying to add some providers, example for WorldOfP2P:
```
jul 26 04:07:28 seedbox systemd[1]: Started Jackett Service.
jul 26 04:07:34 seedbox mono[13972]: 07-26 04:07:34 Info App config/log directory: /usr/share/Jackett/.config/Jackett
jul 26 04:07:35 seedbox mono[13972]: 07-26 04:07:35 Info Starting Jackett 0.7.1594.0
jul 26 04:07:35 seedbox mono[13972]: 07-26 04:07:35 Info Environment version: 4.0.30319.42000 (/usr/lib/mono/4.5)
jul 26 04:07:35 seedbox mono[13972]: 07-26 04:07:35 Info OS version: Unix 4.9.20.1
jul 26 04:07:35 seedbox mono[13972]: 07-26 04:07:35 Info ThreadPool MaxThreads: 400 workerThreads, 200 completionPortThreads
jul 26 04:07:35 seedbox mono[13972]: 07-26 04:07:35 Info issue: Arch Linux \r (\l)
jul 26 04:07:35 seedbox mono[13972]: 07-26 04:07:35 Info mono version: 5.0.0 (Stable 5.0.0.100/9667aa6 Wed Jul 12 18:27:31 UTC 2017)
jul 26 04:07:35 seedbox mono[13972]: 07-26 04:07:35 Info Using HTTP Client: UnixLibCurlWebClient
jul 26 04:08:26 seedbox mono[13972]: 07-26 04:08:26 Info Loading Cardigann definition /opt/Jackett/Definitions/...
...
jul 26 04:08:27 seedbox mono[13972]: 07-26 04:08:27 Info LibCurl init Ok            
jul 26 04:08:27 seedbox mono[13972]: 07-26 04:08:27 Info LibCurl version libcurl/7.54.1 OpenSSL/1.1.0f zlib/1.2.11 libpsl/0.17.0 (+libicu/59.1) libssh2/1.8.0 nghttp2/1.
jul 26 04:08:27 seedbox mono[13972]: 07-26 04:08:27 Info Starting web server at http://*:9117/                                                                          
jul 26 04:12:04 seedbox mono[13972]: 07-26 04:12:04 Info redirecting to /Admin/Dashboard
jul 26 11:47:14 seedbox mono[16099]: 07-26 11:47:14 Info Found 0 releases from WorldOfP2P
jul 26 11:47:14 seedbox mono[16099]: 07-26 11:47:14 Error Exception in test_indexer System.Exception: Found no results while trying to browse this tracker
jul 26 11:47:14 seedbox mono[16099]:   at Jackett.Services.IndexerManagerService+<TestIndexer>c__async0.MoveNext () [0x000ef] in C:\projects\jackett\src\Jackett\Services\IndexerManagerService.cs:201
jul 26 11:47:14 seedbox mono[16099]: --- End of stack trace from previous location where exception was thrown ---
jul 26 11:47:14 seedbox mono[16099]:   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in /build/mono/src/mono-5.0.0/mcs/class/referencesource/mscorlib/system/runtime/exceptionservices/exceptionservicescommon.cs:151
jul 26 11:47:14 seedbox mono[16099]:   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x00037] in /build/mono/src/mono-5.0.0/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:187
jul 26 11:47:14 seedbox mono[16099]:   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in /build/mono/src/mono-5.0.0/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:156
jul 26 11:47:14 seedbox mono[16099]:   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in /build/mono/src/mono-5.0.0/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:128
jul 26 11:47:14 seedbox mono[16099]:   at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00000] in /build/mono/src/mono-5.0.0/mcs/class/referencesource/mscorlib/system/runtime/compilerservices/TaskAwaiter.cs:113
jul 26 11:47:14 seedbox mono[16099]:   at Jackett.Controllers.AdminController+<Test>c__async5.MoveNext () [0x000da] in C:\projects\jackett\src\Jackett\Controllers\AdminController.cs:269
```

**Comment by chibidev** [comment]
Seems to be duplicated by #1597, and the issue with WOP should be fixed by v0.7.1622. Can you name the other trackers affeted? Thanks.

**Comment by chmercesmoreira** [comment]
It's working now, but there is one additional bug with this tracker:
It is not finding any matches for the movie Logan, here is the log from jackett:
```
ul 29 19:59:45 carloshmm.stream mono[25481]: 07-29 19:59:45 Info Found 0 releases from WorldOfP2P for: logan 2017                                                      
jul 29 19:59:47 carloshmm.stream mono[25481]: 07-29 19:59:47 Info Found 0 releases from WorldOfP2P for: logan 2017                                                      
jul 29 19:59:49 carloshmm.stream mono[25481]: 07-29 19:59:49 Info Found 0 releases from WorldOfP2P for: logan noir 2017                                                 
jul 29 19:59:51 carloshmm.stream mono[25481]: 07-29 19:59:51 Info Found 0 releases from WorldOfP2P for: logan noir edition 2017                                         
jul 29 19:59:53 carloshmm.stream mono[25481]: 07-29 19:59:53 Info Found 0 releases from WorldOfP2P for: wolverine 3 2017                                                
jul 29 19:59:55 carloshmm.stream mono[25481]: 07-29 19:59:55 Info Found 0 releases from WorldOfP2P for: wolverine 3 logan 2017                                          
jul 29 20:01:09 carloshmm.stream mono[25481]: 07-29 20:01:09 Info Found 0 releases from WorldOfP2P for: logan 2017                                                      
jul 29 20:01:10 carloshmm.stream mono[25481]: 07-29 20:01:10 Info Found 0 releases from WorldOfP2P for: logan 2017                                                      
jul 29 20:01:13 carloshmm.stream mono[25481]: 07-29 20:01:13 Info Found 0 releases from WorldOfP2P for: logan noir 2017                                                 
jul 29 20:01:15 carloshmm.stream mono[25481]: 07-29 20:01:15 Info Found 0 releases from WorldOfP2P for: logan noir edition 2017                                         
jul 29 20:01:17 carloshmm.stream mono[25481]: 07-29 20:01:17 Info Found 0 releases from WorldOfP2P for: wolverine 3 2017                                                
jul 29 20:01:19 carloshmm.stream mono[25481]: 07-29 20:01:19 Info Found 0 releases from WorldOfP2P for: wolverine 3 logan 2017
```
When I search for these terms directly on jackett It didn't find any matches as well, but I found the reason! If I enclose the string between quotation marks it find matches!
Example:

logan 2017
Showing 0 to 0 of 0 entries

"logan 2017"
Showing 1 to 12 of 12 entries

logan noir 2017
Showing 0 to 0 of 0 entries

"logan noir 2017"
Showing 1 to 1 of 1 entries

Could you please take a look at this?

**Comment by chibidev** [comment]
@chmercesmoreira in case of WoP Jackett is designed to do an additional check when searching for a movie: it filters results from the tracker that do not contain the search keywords. When searching for logan 2017 WoP returns a bunch of uninteresting results, but hardly any of them is related to the actual movie. Thus Jackett filters these.
The query "logan" (without quotation marks) produces the same result as the website of WoP. Whether this additional filtering is required or not could be discussed, however as it is supposed to give you better results, I'd recommend not changing that. The problem is that searching for logan 2017 on WoP does not find Logan (the movie), however that is more likely the site's problem not Jackett's.
For EDIT2, please file a new issue as it is not related to WoP.

**Comment by Guizzoni** [comment]
@chmercesmoreira please create another issue to attend the B2S-Share problem.
Since i created the definition, we can work together to come up with a fix.

**Comment by chmercesmoreira** [comment]
@Guizzoni Are you somehow involved with the Manicomio Share tracker? Jackett is not asking for the login/password for this tracker and I can't login properly without this info.
![screenshot_20170821_181934](https://user-images.githubusercontent.com/20016536/29538721-63f7d1ac-869d-11e7-8946-b1063a96f1ef.png)

**Comment by tnascif** [comment]
@chmercesmoreira just update to pre-release and you'll be fine

**Comment by chmercesmoreira** [comment]
@tnascif
Sorry, I deleted my previous message since I found how to get the cookie:
https://github.com/Jackett/Jackett/wiki/Finding-cookies

**Comment by chmercesmoreira** [comment]
@Guizzoni 
I'm getting the same problem with Manicomio-Share:
```
Aug 21, 2017 06:35:17 PM | Info | Found 0 releases from Manicomio Share for: x men 9 logan 2017
Aug 21, 2017 06:35:14 PM | Info | Found 0 releases from Manicomio Share for: wolverine 3 logan 2017
Aug 21, 2017 06:35:13 PM | Info | Found 0 releases from Manicomio Share for: wolverine 3 2017
Aug 21, 2017 06:35:10 PM | Info | Found 0 releases from Manicomio Share for: logan 2017
Aug 21, 2017 06:35:09 PM | Info | Found 0 releases from Manicomio Share for: logan 2017
Aug 21, 2017 06:34:04 PM | Info | Found 0 releases from Manicomio Share for: logan noir 2017
Aug 21, 2017 06:34:02 PM | Info | Found 0 releases from Manicomio Share for: x men 9 logan 2017
Aug 21, 2017 06:34:00 PM | Info | Found 0 releases from Manicomio Share for: wolverine 3 logan 2017
Aug 21, 2017 06:33:59 PM | Info | Found 0 releases from Manicomio Share for: wolverine 3 2017
Aug 21, 2017 06:33:56 PM | Info | Found 0 releases from Manicomio Share for: logan 2017
Aug 21, 2017 06:33:54 PM | Info | Found 0 releases from Manicomio Share for: logan 2017
```

Could you please take a look at this?

**Comment by Guizzoni** [comment]
@chmercesmoreira you're using an old Jackett version which contains definitions that didn't work properly. Please update to the latest Jackett pre-release, re-add "Manicômio Share" and try again.

**Comment by Guizzoni** [comment]
Also, the B2S-Share issue you were having, should be fixed on the latest pre-release.

**Comment by chmercesmoreira** [comment]
@Guizzoni 
Yes you are right, after using the pre-release version the problem is gone.

But now there is one new problem: radarr is grabbing the releases correctly now but every match has no quality (all is set to unknown).

Should I open a new issue? Or this is a particularity of macomio share?

**Comment by Guizzoni** [comment]
Not sure... I'm using Sonarr and CouchPotato, and both of them shows the right quality for every release. Maybe it's something in Radarr?

As for the age, yes, it is a known issue with the "Manicomio" tracker, that's because of the way the tracker shows the date on the releases (only after pressing the + button) so it's not possible to grab the correct date in Jackett, as far as i know.

**Comment by chmercesmoreira** [comment]
@Guizzoni 
Sonarr is showing the quality properly, only the age has the same problem as you said:
![screenshot_20170821_204050](https://user-images.githubusercontent.com/20016536/29542372-1709145a-86b1-11e7-9c04-a79d0f6a7a11.png)

I don't know how to fix radarr, I already deleted it, completly purged it's configurations files but the problem remain. Are you sure that radarr is showing the quality using the tracker manicomio share?

**Comment by Guizzoni** [comment]
Yeah, the age is something I can't fix unfortunately. 

I don't use Radarr so I don't know about that. But since Sonarr and CouchPotato are working, i think this issue is related to Radarr.

**Comment by chmercesmoreira** [comment]
Ok, thank you.

**Comment by chmercesmoreira** [comment]
@Guizzoni 
I just tested the pirate bay tracker and the quality of all the results were grabbed correctly. So I think this is a problem specific to Manicomio Share. Should I open a new issue here ou should I open a new issue on Radarr github?

**Comment by Guizzoni** [comment]
This is something related to Radarr not detecting the quality on the Manicomio Share release titles. So i think you should open a new issue on Radarr github.
